cloud/ec2_util.go

```diff
@@ -414,6 +415,17 @@ func makeBlockDeviceMappings(mounts []MountPoint) ([]*ec2aws.BlockDeviceMapping,
 			if mount.VolumeType != "" {
 				m.Ebs.VolumeType = aws.String(mount.VolumeType)
 			}
+			if mount.Throughput != 0 {
+				//aws only allows values between 125 and 1000
+				if mount.Throughput > 1000 || mount.Throughput < 125 {
+					return nil, errors.New("throughput must be between 125 and 1000")
+				}
+				// This parameter is valid only for gp3 volumes.
+				if utility.FromStringPtr(m.Ebs.VolumeType) != ec2aws.VolumeTypeGp3 {
+					return nil, errors.New("throughput is only valid for gp3 volumes")
+				}
+				m.Ebs.Throughput = aws.Int64(mount.Throughput)
```

Comment:
Can we add this validation and the 125 - 1000 check to the distro page? otherwise it's easy to save values that will break the distro https://github.com/evergreen-ci/evergreen/blob/90cb3e180815e3b4b54bde99bc2e008ddc647a41/validator/distro_validator.go#L24

Reply:
I don't think I can add it here because the distro doesn't store the ebs device information, and there doesn't seem to be a volume validator.

Reply:
It is part of distro; it's part of the EC2 provider settings. Go isn't great at interfaces so instead we stored this as a [list of birch documents](https://github.com/evergreen-ci/evergreen/blob/cf637f34e4a5c031cb3638046a252c76d02f19b9/model/distro/distro.go#L30), and unmarshal [based on provider](https://github.com/evergreen-ci/evergreen/blob/4a7a32041d8bd02f4784321820eb2655f7476df2/cloud/cloud.go#L238).  It looks like we do validate the EC2 provider settings. 
That being said, it looks like we do already call makeBlockDeviceMappings when we validate the EC2 settings. https://github.com/evergreen-ci/evergreen/blob/877d8c75a4cdc40e12f5faa4f6d3ceca617fbadd/cloud/ec2.go#L107 
which is called in [this validator](https://github.com/evergreen-ci/evergreen/blob/90cb3e180815e3b4b54bde99bc2e008ddc647a41/validator/distro_validator.go#L136) so we should be good. Can you verify on local if this prevents you from saving the page?